code42_connector.py

```diff
@@ -31,8 +31,11 @@ def wrapper(f):
     return wrapper
 
 
-class Code42Connector(BaseConnector):
+def _convert_to_obj_list(scalar_list, sub_object_key="item"):
```

Comment:
maybe the second arg should be required for this method. I'm not sure `items` frequently enough for it to be considered a default for this, and making it required would force someone to think more closely about it and make it less likely for them to erroneously use `items`.

Reply:
nvm I misunderstood what this method does, this makes sense now